addons/l10n_ec/models/account_tax_group.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+class AccountTaxGroup(models.Model):
+    _inherit = 'account.tax.group'
+
+    _TYPE_EC = [
+        ('vat12', 'VAT 12%'),
+        ('vat14', 'VAT 14%'),

```

Comment:
Because???
Only must be VAT.  for identify tax amount, must be done into account.tax(field amount), not in account.tax.group

If in future SRI add more taxes related to IVA, this field is complex to maintenance

Reply:
@celm1990  the main reason for this is to be compatible with tax grouping of Odoo which then automatically creates subtotals in invoices.

Last time the SRI requested vat 12% and vat 14% they requested it to be shown in separate subtotal fields, then it is likely that next time the SRI creates a vat different than 12% they will require it to be shown in separate subtotal fields in the invoice. That's why it is necesary to have two tax groups for automatic subtotal grouping in different subtotal fields